ipi/engine/motion/replay.py

```diff
@@ -58,7 +61,34 @@ def __init__(self, fixcom=False, fixatoms=None, intraj=None):
             raise ValueError(
                 "Replay can only read from PDB or XYZ files -- or a single frame from a CHK file"
             )
-        self.rfile = open(self.intraj.value, "r")
+        # Posibility to read beads from separate XYZ files by a wildcard
+        if any(char in self.intraj.value for char in "*?[]"):
+            infilelist = []
+            for file in sorted(os.listdir('.')):
+                if fnmatch(file, self.intraj.value):
+                    infilelist.append(file)
+            # determine bead numbers in input files
+            bead_map_list = []
+            for file in infilelist:
+                fdin = open(file, 'r')
+                rr = read_file_raw(self.intraj.mode, fdin)
+                metainfo = rr['comment'].split()
+                for i, word in enumerate(metainfo):
+                    if word == 'Bead:':
+                        bead_map_list.append(int(metainfo[i + 1]))
+                fdin.close()
+            # check that beads are continuous (no files missing)
+            if sorted(bead_map_list) != list(range(len(bead_map_list))):
```

Comment:
Why should this be an error? What if I want to calculate the quantities of only a few selected beads. Can you comment on this?

Reply:
I thought conservatively. In principle, this is not necessary, but I would leave at least a warning, because people can store files with similar names in the same folder, and they can accidentally match the pattern. E.g. `simulation.pos_00.xyz` and `simulation.pos_00.old.xyz` both match `simulation.pos_*.xyz`.

Reply:
Changed error to warning